.travis.yml

```diff
@@ -27,7 +28,9 @@ matrix:
   - php: 5.2
     dist: precise
   - php: 7.2
-    env: WP_VERSION=latest WP_MULTISITE=0 RUN_PHPCS=1 RUN_E2E=1
+    env: WP_VERSION=latest WP_MULTISITE=0 RUN_PHPCS=1
```

Comment:
This gets confusing in the Travis output when glancing, I would use `name` config property to include a description of the test. E.g. like in wc-admin: https://github.com/woocommerce/wc-admin/blob/7cd132211609cc4ecf3cf47d211c6905e1d3b52a/.travis.yml#L7

Reply:
That is great. I didn't know about the `name` config property. I will create a new commit to add it. Thanks.

Reply:
Commit 87e47cd adds the `name` config property as suggested to identify the build jobs used to run unit tests code coverage, PHPCS and e2e tests.